azext_iot/_params.py

```diff
@@ -104,11 +104,15 @@ def load_arguments(self, _):
             validator=mode2_iot_login_handler,
             help="This command supports an entity connection string with rights to perform action. "
             'Use to avoid session login via "az login". '
-            "If both an entity connection string and name are provided the connection string takes priority.",
+            "If both an entity connection string and name are provided the connection string takes priority."
```

Comment:
nit, lets put a space between `"Required if --hub-name is not provided."` and the prior sentence.